demo/admin/src/common/MasterMenu.tsx

```diff
@@ -158,6 +159,15 @@ export const masterMenuData: MasterMenuData = [
                 },
                 requiredPermission: "pageTree",
             },
+            {
+                type: "route",
+                primary: <FormattedMessage id="menu.project-snips.formBuilder" defaultMessage="Form Builder" />,
+                route: {
+                    path: "/project-snips/form-builder",
+                    component: FormBuilderPage,
+                },
+                requiredPermission: "formBuilders",
+            },
```

Comment:
IMO this should be in structured content.

Reply:
The design showed this under "Project Snips" 🤷🏼‍♂️